Mathlib/Topology/Algebra/InfiniteSum/NatInt.lean

```diff
@@ -335,3 +272,222 @@ theorem Summable.tendsto_atTop_zero {f : ℕ → G} (hf : Summable f) : Tendsto
 #align summable.tendsto_at_top_zero Summable.tendsto_atTop_zero
 
 end TopologicalGroup
+
+end Nat
+
+/-!
+## Sums over `ℤ`
+
+In this section we prove a variety of lemmas relating sums over `ℕ` to sums over `ℤ`.
+-/
+
+section Int
+
+section Monoid
+
+lemma HasSum.natCast_add_negSucc {f : ℤ → M} (hf : HasSum f m) :
+    HasSum (fun n : ℕ ↦ f n + f (Int.negSucc n)) m := by
```

Comment:
I'm not sure whether `Int.negSucc n` is really the normal form we want to use -- it feels more like an implementation detail of the integers in Lean. What about using `- (n+1)` instead (which is defeq)? Note I'm not sure of myself here, as I don't think there is a simp lemma converting the former to the latter, so it might be worth asking on Zulip what people think.

Reply:
I actually had the same feeling, but didn’t want to unilaterally change the existing formulations. Since you don’t like the negSucc formulations either, I have pushed a change.

> I'm not sure whether `Int.negSucc n` is really the normal form we want to use -- it feels more like an implementation detail of the integers in Lean. What about using `- (n+1)` instead (which is defeq)? Note I'm not sure of myself here, as I don't think there is a simp lemma converting the former to the latter, so it might be worth asking on Zulip what people think.